Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "JutulDarcyRules"
 uuid = "41f0c4f5-9bdd-4ef1-8c3a-d454dff2d562"
 authors = ["Ziyi Yin <ziyi.yin@gatech.edu>"]
-version = "0.2.5"
+version = "0.2.7"
```

Comment:
Julia usually complains and fails to register the new version if release are not consecutive

Reply:
Ah ok. I wanted to keep up with JutulDarcy version, but I'll drop back down to 0.2.6.